CHANGELOG.rst

```diff
@@ -30,6 +30,7 @@ Since last release
 * Created Package class and optional declaration of packages in input files (#1673, #1699), package id is a member of resources (materials/products) (#1675)
 * CI support for Rocky Linux (#1691)
 * Added support for a ResBuf to behave as a single bulk storage with mixing & extraction of resources (#1687)
+* Removed deprecated `smbchk.py` functionality and tests (#1706)
```

Comment:
Maybe if we add a few more words to the changelog, it will leave a trail of where to find the code again, and we can continue with removing this code.
```suggestion
* Removed deprecated `smbchk.py` ABI consistency checking functionality and tests (#1706)
```